src/utils.ts

```diff
@@ -204,9 +203,16 @@ const isString = (value: any): boolean => typeof value === 'string';
 const isNumber = (value: any): boolean => typeof value === 'number';
 const isFunction = (fn: any): boolean => typeof fn === 'function';
 
-// TODO: Refactor this to a regex
-const isDataPlanSlug = (str: string): boolean =>
-    isString(str) && str === slugify(str);
+const toSlug = (value: any): string =>
```

Comment:
Technically these are not slugs, because slugs can have hyphens.  I think we should really call this something more specific to our purposes:

```suggestion
const toDataPlanId = (value: any): string =>
```